ksql-engine/src/test/resources/query-validation-tests/group-by.json

```diff
@@ -17,12 +17,43 @@
         {"topic": "test_topic", "key": "d1", "value": "d1"}
       ],
       "outputs": [
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d1", "value": "d1,0,1"},
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d2", "value": "d2,0,1"},
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d1", "value": "d1,0,2"},
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d2", "value": "d2,0,2"},
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d1", "value": "d1,0,3"},
         {"topic": "OUTPUT", "key": "d1", "value": "d1,1"},
         {"topic": "OUTPUT", "key": "d2", "value": "d2,1"},
         {"topic": "OUTPUT", "key": "d1", "value": "d1,2"},
         {"topic": "OUTPUT", "key": "d2", "value": "d2,2"},
         {"topic": "OUTPUT", "key": "d1", "value": "d1,3"}
       ]
+    },{
+      "name": "field (stream->table) - format",
+      "statements": [
+        "CREATE STREAM TEST (data VARCHAR) WITH (kafka_topic='test_topic', KEY='data', value_format='{FORMAT}');",
+        "CREATE TABLE OUTPUT AS SELECT data, COUNT(*) FROM TEST GROUP BY DATA;"
+      ],
+      "format": ["AVRO", "JSON"],
+      "inputs": [
+        {"topic": "test_topic", "key": "d1", "value": {"DATA": "d1"}},
+        {"topic": "test_topic", "key": "d2", "value": {"DATA": "d2"}},
+        {"topic": "test_topic", "key": "d1", "value": {"DATA": "d1"}},
+        {"topic": "test_topic", "key": "d2", "value": {"DATA": "d2"}},
+        {"topic": "test_topic", "key": "d1", "value": {"DATA": "d1"}}
+      ],
+      "outputs": [
+        {"topic": "_confluent-ksql-some.ksql.service.idquery_CTAS_OUTPUT_0-Aggregate-aggregate-changelog", "key": "d1", "value": {"KSQL_INTERNAL_COL_0": "d1", "KSQL_INTERNAL_COL_1": 0, "KSQL_AGG_VARIABLE_0": 1}},
```

Comment:
For my own understanding, what is this `KSQL_INTERNAL_COL_1` that's appearing in the aggregation topic (and always has value 0)?

Reply:
`ROWTIME`.  Which we obviously don't need in there, given the same timestamp is available on the record produced to Kafka.  But it's there...

Reply:
Weird... surprised it's not named `ROWTIME` like in some of the other internal topics down below.